tests/blockchain/regtest/RoundRobinRecaptureTransactions.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ottest/fixtures/blockchain/RegtestSimple.hpp"
+#include "ottest/fixtures/paymentcode/VectorsV3.hpp"
+
+#include <gtest/gtest.h>
+#include "opentxs/util/Log.hpp"
+
+#include <chrono>
+#include <thread>
+
+namespace ottest
+{
+
+TEST_F(Regtest_fixture_simple, round_robin_recapture_transactions)
+{
+    EXPECT_TRUE(Start());
+    EXPECT_TRUE(Connect());
+
+    const auto blocks_number = 2;
+    auto coin_to_send =
+        blocks_number * transaction_in_block_ * amount_in_transaction_ -
+        1000000;
+    auto instance = 2;
+    {
+        auto [user_core, success_core] = CreateClient(
```

Comment:
hmm honestly, I do not see much difference between core and auxiliary wallets in this test. Can you remind me what was the difference?